src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectOptionsProcessor.cs

```diff
@@ -31,15 +33,14 @@ internal class VisualStudioProjectOptionsProcessor : IDisposable
         /// as a low cost (in terms of memory) way to determine if the command line
         /// actually changes and we need to make any downstream updates.
         /// </summary>
-        private Checksum? _commandLineChecksum;
+        private Checksum _commandLineChecksum;
 
         /// <summary>
         /// To save space in the managed heap, we dump the entire command-line string to our
         /// temp-storage-service. This is helpful as compiler command-lines can grow extremely large
         /// (especially in cases with many references).
         /// </summary>
-        /// <remarks>Note: this will be null in the case that the command line is an empty string.</remarks>
```

Comment:
@CyrusNajmabadi this is undoing the optimization you did in e5c2dea2f74ce7819996389693769c76ef9ed2f2 -- is that a concern?

Reply:
checking.